src/test/java/gov/va/viers/cdi/emis/MilitaryInfoTests.java

```diff
@@ -35,11 +35,30 @@
   @Autowired private DoDAdapterClient dodClient;
 
   public void initSuccessMock() {
+    String sampleResponsePath = "exampleSuccessVadirResponse_MilInfoEligSvc.xml";
+    JAXBElement<EMISmilitaryServiceEligibilityResponseType> jaxbDodResponse =
+        getSampleJaxbDodResponse(sampleResponsePath);
+    Mockito.doReturn(jaxbDodResponse)
+        .when(dodClient)
+        .getMilitaryServiceEligibilityResponse("6001010072");
+  }
+
```

Comment:
Consider pulling out initSuccessMock and initBadFormatMock into an initMock method that takes in a sampleResponsePath and an EDIPIValue.